modules/geotiff/src/lib/tiff-pixel-source.ts

```diff
@@ -26,6 +26,11 @@ class TiffPixelSource<S extends string[]> implements PixelSource<S> {
     public meta?: PixelSourceMeta
   ) {
     this._indexer = indexer;
+    this.dtype = dtype;
```

Comment:
Remove the `public` keyword from the parameters and declare the members on line 17.

Reply:
Done